source/system/math.ooc

```diff
@@ -33,6 +35,8 @@ extend Long {
 	}
 	maximum: func (other: This) -> This { this > other ? this : other }
 	minimum: func (other: This) -> This { this < other ? this : other }
+	maximumValue ::= static INT64_MAX
+	minimumValue ::= static INT64_MIN
```

Comment:
Sort of breaks the point of the general policy though, but alright I don't mind.